modules/oadp-pod-crash-set-resource-request-restic.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-pod-crash-resource-request-retics_{context}"]
+= Setting resource requests for a Restic pod
+
+You can use the `configuration.restic.podConfig.resourceAllocations` specification field to set specific resource requests for a `Restic` pod.
+
+.Procedure
+
+* Set the `cpu` and `memory` resource requests in the `yaml` file:
+
+.Example Restic file
+
+[source,yaml]
+----
+  configuration:
```

Comment:
There's extra indentation here; Which API is this for?

No one follows the API guidelines, but nonetheless, ideally we're including the following information for every API so users know what is going on at a glance:

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting-for-kubernetes-and-openshift-api-objects

Thanks.